safegraph/tests/test_process.py

```diff
@@ -62,5 +68,47 @@ def test_handle_wip_signal(self):
         assert signal_names[0].startswith("wip_")
         assert all(not s.startswith("wip_") for s in signal_names[1:])
 
+    def test_files_in_past_week(self):
```

Comment:
Are failure cases tested too? (e.g., missing date files)

Reply:
This function does not do any testing of file existence.  I should add file existence testing to `process()`

Reply:
Cool -- yeah because it looks like this is new logic it's hard to tell if this introduces any new edge cases so would suggest thorough testing if it is not there already.

Reply:
Added more test cases across year boundary and leap day checking